server/data/restClient.ts

```diff
@@ -103,6 +103,37 @@ export default class RestClient {
     }
   }
 
+  async put({
+    path = null,
+    headers = {},
+    responseType = '',
+    data = {},
+    raw = false,
+  }: PostRequest = {}): Promise<unknown> {
```

Comment:
Should this be a `PutRequest`?

Reply:
So we basically want a type that has the same properties as `PostRequest` but is called `PutRequest` but the linter says we can't extend a type without changing the properties... I think it's probably best to just disable the linter though

Reply:
Aaah - maybe give the type a more generic name like `PostOrPutRequest`?

Reply:
I've opted to turn the lint rule off - I don't think it's useful